### DISCUSSION TITLE
Addition to LearnJS

### DISCUSSION THREAD

**Discussion by imjustadonut** [sanskaarz/LearnJS#10] [Show and tell]
I added a few extra files if u wanna check out my repo and copy them over. Check them out! Its a fork from yours.
Also, I emailed you at your contact email on your ReadMe.md about something.

**Comment by sanskaarz** [comment]
Hey @imjustadonut,
Thanks for contributing but could you please **raise a pull request**,
because I am unable to see any email your side and the changes  in yours forked repo.

**Reply by imjustadonut** [reply]
Hi! Yeah. Can you check your email? I was wondering about something.

**Reply by sanskaarz** [reply]
Are you sending it to sanskar786g@gmail.com?
because I still haven't received it.

**Reply by imjustadonut** [reply]
Let me check.

**Reply by sanskaarz** [reply]
Hey got your mail!!!

**Reply by sanskaarz** [reply]
It was in spam section. idk why

**Reply by imjustadonut** [reply]
:D
